### PR TITLE
fix(basetest): Remove duplicate exception step trace logging

### PR DESCRIPTION
**Pull request by okurz**
Motivation:
When an exception occurs (such as a timeout in assert_screen), the
stack trace is currently logged twice with an incremented step counter.
This creates a confusing, duplicated error condition in the openQA UI.

Design Choices:
Removed the second redundant call to bmwqemu::update_line_number in the
exception catch block of runtest(). Fixed t/08-autotest.t to expect the
stepinfo log to now appear before the stack trace instead of after.
Also added missing SERIAL_CONSOLE_SILENCE_TIMEOUT documentation to fix
test failures.

### CODE REVIEW

**Comment by perlpunk** [comment]
I don't understand. Can you link an example test?

File: doc/backend_vars.md [hunk 1]
```diff
@@ -2,6 +2,7 @@ Supported variables per backend
 ===============================
 
 ## COMMON backend
+| SERIAL_CONSOLE_SILENCE_TIMEOUT | integer | 300 | Configures the timeout in seconds for which a serial console SSH channel can remain silent before a warning diagnostic is emitted during wait_serial |
```
Comment (perlpunk) [inline doc/backend_vars.md]: How is that related?

**Review by perlpunk — CHANGES_REQUESTED** [review]
If you mean the stack trace in the log that links to the source code, that needs to be there for the step information, so the link in the test details step box doesn't show an empty box. every box should have an according entry in autoinst-log.txt.
Alternative suggestions welcome

File: doc/backend_vars.md (hunk 1 above)
Comment (okurz) [reply doc/backend_vars.md]: wait, that's the wrong commit, sorry. fixed.

File: basetest.pm [hunk 2]
```diff
@@ -353,7 +353,6 @@ sub runtest ($self) {
         if (!$internal && $error_message =~ /Can't locate .+ in \@INC/) {
             my $msg = "# Test died with missing dependency: $e";
             bmwqemu::fctinfo($msg);
-            bmwqemu::update_line_number();
```
Comment (perlpunk) [inline basetest.pm:356]: I can imagine calling a different function here that only prints the category/module/step marker, but not the stack trace. Then the links from the details view would still work.
WDYT?

Comment (okurz) [reply basetest.pm:356]: good idea

**Comment by okurz** [comment]
I couldn't find the test results anymore which led me to this approach. I guess we don't need it.
